pkg/controllers/nsautoprop/controller.go

```diff
@@ -255,6 +255,8 @@ func (c *Controller) reconcile(ctx context.Context, qualifiedName common.Qualifi
 		return worker.StatusAllOK
 	}
 
+	c.recordNsAutoPropError(fedNamespace)
```

Comment:
what exactly are we trying to measure here? this only records the sync failures of namespace objects. if that's the case, would a corresponding metric emitted by the sync controller suffice?

Reply:
yes, only records the sync failures of namespace objects.

Reply:
other resources may not have this requirement.